lib/webpacker/railtie.rb

```diff
@@ -12,6 +12,7 @@ class Webpacker::Engine < ::Rails::Engine
       include Webpacker::Helper
     end
 
+    Webpacker.logger = ::Rails.logger
```

Comment:
I think you have to set this in a config.after_initialize style, like we do with Active Storage: https://github.com/rails/rails/blob/master/activestorage/lib/active_storage/engine.rb#L16